components/NavigationBar/src/MDCNavigationBar.h

```diff
@@ -131,6 +131,20 @@ IB_DESIGNABLE
  */
 @property(nonatomic, strong, null_resettable) UIFont *titleFont;
 
+/**
+ A behavioral flag that affects whether titleFont can be set to a font of any size or not.
+
+ If enabled, titleFont can be set to a font of any size.
+
```

Comment:
These are intentionally separate so that it's clear that one paragraph is describing an enabled state, while the other is describing the disabled state.